tvault/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source ~/.nvm/nvm.sh
+cd opt && git clone -b tvault_docker https://github.com/rajeevr2715/t-vault && cd t-vault &&./build_tvault.sh --build all --package tar && sudo ./install_tvault.sh ${ADMIN_PASS} ${CONSUL_STORAGE_ADDRESS}
```

Comment:
Are we ready to use the actual repo instead of your fork?

Reply:
No. Actual repo supports ECS fargate and latest version related changes. But in our case,
1. We need to set our own user password. Actual repo wont support that. https://github.com/rajeevr2715/t-vault/blob/7829936632f16f4492a99d27d13c0974317cdefe/dist/src/main/docker/vault.sh#L30
2. We need to pass consul address in the parameter. We made that to docker environment variable. https://github.com/rajeevr2715/t-vault/blob/7829936632f16f4492a99d27d13c0974317cdefe/dist/src/main/docker/vault.sh#L31
In actual repo, the default is File system and we have to manually/SED the consul adress.

https://github.com/rajeevr2715/t-vault/blob/7829936632f16f4492a99d27d13c0974317cdefe/install_tvault.sh

3. We are disabling the https option in tvault. https://github.com/rajeevr2715/t-vault/blob/7829936632f16f4492a99d27d13c0974317cdefe/dist/src/main/components/web/nginx/conf/nginx.conf#L31

Since the docker image build is internal, we can move this folder into an S3 or somewhere else in AWS. Please advice